File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    n = get_int("Number: ")
    if n >= 1 and n <= 8:
        break
for i in range(1, n + 1):
    for j in range(n):
        if j < n - i:
            print(" ", end="")
        else:
            print("#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void drawPyramid(void);

int main(void)
{
    drawPyramid();
}
void drawPyramid(void)
{
    int size;
    do
    {
        size = get_int("Height: ");
    }
    while (size < 1 || size > 8);

    // printf("Stored: %d",size);

    for (int i = 1; i <= size; i++)
    {
        for (int j = 0; j < size; j++)
        {
            if (j < size - i)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}
